### ISSUE TITLE
[Ninja] Shadows not applying on cast

### ISSUE THREAD

**Issue by kaincenteno** [DerpyProjectGroup/topaz#1789]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`

## Steps to reproduce

Casting utsusemi ichi or ni
no shadows effect added

```
[05/23/22 16:09:27:433][map][error][error] luautils::onSpellCast: ./scripts/globals/spells/spell_enhancing_ninjutsu.lua:58: attempt to index field 'spell_enhancing_Ninjutsu' (a nil value)
stack traceback:
        ./scripts/globals/spells/spell_enhancing_ninjutsu.lua:58: in function <./scripts/globals/spells/spell_enhancing_ninjutsu.lua:45> (OnSpellCast:2519)
```

## Expected behavior

casting utsusemi ichi or ni
gets shadows effect on char

**Comment by kaincenteno-doordash** [comment]
Close this if resolved by https://github.com/LandSandBoat/server/pull/1790

**Comment by CatsEyeXI** [comment]
this was resolved for us by #1790

**Comment by AikoSachi** [comment]
Additional confirmation that after pushing this fix to my server the shadows issue has been fixed.
